platforms/android/build_sdk.py

```diff
@@ -264,6 +265,9 @@ def build_library(self, abi, do_install, no_media_ndk):
         if no_media_ndk:
             cmake_vars['WITH_ANDROID_MEDIANDK'] = "OFF"
 
+        if len(self.disable) > 0:
+            cmd.extend(["-DWITH_%s=OFF" % f for f in self.disable])
```

Comment:
Use `cmake_vars` instead.

May be even after this line: `cmake_vars.update(abi.cmake_vars)`